### PR TITLE
PER-9159: Back-end: write read endpoint for directives

### PR DESCRIPTION
**Pull request by liam-lloyd**
This is build atop the PER-9148 PR, so if that isn't merged yet compare to `per-9148` to see what's new here.

### CODE REVIEW

**Comment by cecilia-donnelly** [comment]
I think [6a65abb](https://github.com/PermanentOrg/stela/pull/14/commits/6a65abb083a7e210de6aaf3c4d76dd4ddf7c8637) is the only relevant commit here and the other two are leftover from old versions of the parent branch(es).

**Review by cecilia-donnelly — CHANGES_REQUESTED** [review]
It's possible I've erred in my token management, but I believe that this allows a logged-in user to view the directives associated with *any* archive, which is not what we want. I think a regular user should only be able to see directives associated with archives they own.

(There's maybe a question about an admin being able to see all directives on all archives, but I don't think we've designed that yet.)

File: src/directive/service/get_by_archive_id.test.ts
Comment (cecilia-donnelly) [inline src/directive/service/get_by_archive_id.test.ts:43]: Probably want a "should return empty list if authenticated account does not own archive," assuming that we don't want to confirm a directive exists (i.e., essentially return a 404 rather than a 401)

**Review by cecilia-donnelly — COMMENTED** [review]
This is missing some fixes from main, I believe because it includes old versions of the "post a directive" and "trigger a directive" commits.

Other than that this behaves as I expect and looks good! I'll officially approve once those other commits are gone/fixed up / whatever needs to happen.

**Comment by liam-lloyd** [comment]
> (Are the package lock changes necessary?)

Oh, I imagine not, since package.json hasn't changed. I'll remove those changes before merging.

**Pull request merged**
